vehicles.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS `player_vehicles` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `license` varchar(50) NOT NULL,
+    `citizenid` varchar(50) NOT NULL,
+    `vehicle` varchar(50) DEFAULT NULL,
```

Comment:
Could we put a comment next to this? I think vehicle is supposed to be the vehicle name right? Is this the same as the model name? Maybe we don't need to store both hash and name then and can deprecate one of these columns?

Reply:
Do we know of any resource that used the hash over the vehicle(model)? Will deprecating hash be an issue?

Reply:
What we can do is write to both but only read from one. That way we pick one as the source of truth but maintain the same schema